pyproject.toml

```diff
@@ -70,7 +70,6 @@ mkdocs = "^1.4.2"
 mkdocs-material = "^9.0.8"
 mkdocstrings-python = "^0.8.3"
 mike = "^1.1.2"
-python-semantic-release = {git="https://github.com/tumaysem/python-semantic-release.git"}
```

Comment:
Because of this, `poetry.lock` has to be updated using `poetry lock --no-update`.